kudos-webapps/src/main/webapp/vue-app/kudos/components/KudosApp.vue

```diff
@@ -6,111 +6,104 @@
-                  <kudos-identity-link
-                    :id="kudos.receiverId"
-                    :technical-id="kudos.receiverIdentityId"
-                    :type="kudos.receiverType"
-                    :name="kudos.receiverFullName" />
-                </v-card-text>
-                <!-- The same block is displayed again because the first block is absolute, so this is to ensure that the element is displayed in its correct position -->
-                <v-card-text v-else-if="kudos.receiverFullName" class="kudosIconLink kudosIconLinkInvisible">
-                  <kudos-identity-link
-                    :id="kudos.receiverId"
-                    :technical-id="kudos.receiverIdentityId"
-                    :type="kudos.receiverType"
-                    :name="kudos.receiverFullName" />
-                </v-card-text>
-                <v-card-text v-else class="px-0 py-0">
-                  <a :title="$t('exoplatform.kudos.label.remainingKudos', (remainingKudos -1))" href="javascript:void(0);">X {{ remainingKudos - 1 }}</a>
-                </v-card-text>
-              </v-card>
-            </v-layout>
-          </v-container>
-          <div v-if="remainingKudos <= 0" class="alert alert-info mt-5">
-            <i class="uiIconInfo"></i>
-            {{ $t('exoplatform.kudos.info.noKudosLeft', {0: remainingDaysToReset, 1: remainingDaysToReset === 1 ? $t('exoplatform.kudos.label.day') : $t('exoplatform.kudos.label.days') }) }}
+      @opened="drawer = true"
+      @closed="drawer = false"
+      :temporary="temporaryDrawer"
+      id="activityKudosDrawer"
+      allow-expand
+      right
+      disable-pull-to-refresh>
+      <template slot="title">
+        <span class="text-capitalize-first-letter">
+          {{ $t('exoplatform.kudos.title.sendAKudos') }}
+        </span>
+      </template>
+      <template slot="content">
+        <div
+          ref="activityShareFrom"
+          class="flex mx-4 pt-6">
+          <div class="d-flex flex-column flex-grow-1">
+            <div class="d-flex flex-row">
+              <span class="text-header-title my-auto">{{ $t('exoplatform.kudos.content.to') }} </span>
+              <div class=" flex-grow-1 ">
+                <div class="d-flex flex-column flex-grow-1 pt-6 ">
+                  <div class="d-flex flex-row">
+                    <exo-user-avatar
+                      :username="kudosReceiver.receiverId"
+                      :fullname="kudosReceiver.fullName"
+                      :avatar-url="kudosReceiver.avatar"
+                      :url="kudosReceiver.profileUrl"
+                      bold-title
+                      link-style
+                      size="32"
+                      class="mx-4  my-auto" />
+                  </div>
+                  <div class="d-flex flex-row">
+                    <span class=" mx-4 grey--text" style="font-size:80%">
+                      {{ $t('exooplatform.kudos.label.numberOfKudos', {0: kudosSent}) }}
```

Comment:
We need to use the variable stocked in `window.kudosSettings.kudosPeriodType` to show the period and `window.kudosSettings.kudosPerPeriod` to show the kudos number allowed per period.